packages/utils/src/packages/load-resource.ts

```diff
@@ -0,0 +1,303 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as globby from 'globby';
+import * as semver from 'semver';
+
+import { cwd, loadJSONFile } from '../fs';
+import { debug as d } from '../debug';
+import { isFullPackageName } from './is-full-package-name';
+import { ResourceType } from '../types/resource-type';
+import { loadPackage as getPackage } from './load-package';
```

Comment:
Since we renamed the conflicting `loadPackage` to `requirePackage`, let's just drop this alias to `getPackage` and use the original helper name.